src/PIL/FitsImagePlugin.py

```diff
@@ -75,26 +59,13 @@ def _open(self):
             self.mode = "F"
             # rawmode = "F" if number_of_bits == -32 else "F;64F"
 
-        self.fp.seek(offset)
-
-        loader = self._load()
-        if loader:
-            loader.open(self)
-
-    def _load(self):
-        return _handler
-
-
-def _save(im, fp, filename):
-    if _handler is None or not hasattr("_handler", "save"):
-        raise OSError("FITS save handler not installed")
-    _handler.save(im, fp, filename)
+        offset = math.ceil(self.fp.tell() / 2880) * 2880
```

Comment:
The documentation mentioned that the offset for the image data was a multiple of 2880.

https://fits.gsfc.nasa.gov/fits_primer.html
> Each header or data unit is a multiple of 2880 bytes long. If necessary, the header or data unit is padded out to the required length with ASCII blanks or NULLs depending on the type of unit.

> The data unit, if present, immediately follows the last 2880-byte block in the header unit.